lib/process-services-cloud/src/lib/task/task-filters/components/task-filters-cloud.component.ts

```diff
@@ -31,6 +31,13 @@ import { TaskCloudEngineEvent } from '../../../models/engine-event-cloud.model';
     styleUrls: ['base-task-filters-cloud.component.scss']
 })
 export class TaskFiltersCloudComponent extends BaseTaskFiltersCloudComponent implements OnInit, OnChanges {
+
+    static WS_NOTIFICATIONS_KEY = 'notifications.task-filters';
+
+    /** Enable Bubble Notifications on Task Filter Count */
+    @Input()
+    enableNotifications: boolean;
```

Comment:
Do we really need this input? From my point of view, either is the parent to check the app config.
```this.enableNotifications = this.appConfigService.get(TaskFiltersCloudComponent.WS_NOTIFICATIONS_KEY, false);```
and pass the information down or the component itself